### DISCUSSION TITLE
Is this behavior intended?

### DISCUSSION THREAD

**Discussion by RomanGlazkov** [GrapesJS/grapesjs#4970] [Q&A]
Hi there!
Please look at the demo:
[Screencast from 10.03.2023 18:41:02.webm](https://user-images.githubusercontent.com/66161402/224360248-d0b1790a-8f2f-497c-9c14-b87558939ac0.webm)

@artf  Is it okay behavior or is it a bug? Thank you in advance

**Comment by artf** [comment]
This seems to be related to the native contentEditable behaviour as it tries to create nested links
